src/coreclr/jit/importer.cpp

```diff
@@ -10458,6 +10458,17 @@ void Compiler::impLoadArg(unsigned ilArgNum, IL_OFFSET offset)
         {
             lclNum = lvaArg0Var;
         }
+#ifdef SWIFT_SUPPORT
+        else if (lclNum == lvaSwiftErrorArg)
+        {
+            // Convert any usages of the SwiftError* out parameter to pointers to the SwiftError pseudolocal
+            assert(info.compCallConv == CorInfoCallConvExtension::Swift);
+            assert(lvaSwiftErrorLocal != BAD_VAR_NUM);
+            impPushOnStack(gtNewLclVarAddrNode(lvaSwiftErrorLocal, TYP_I_IMPL), typeInfo(TYP_I_IMPL));
```

Comment:
```suggestion
            assert((lvaSwiftErrorLocal != BAD_VAR_NUM) && (lvaSwiftErrorArg != BAD_VAR_NUM));
            var_types type = lvaGetDesc(lvaSwiftErrorArg)->TypeGet();
            impPushOnStack(gtNewLclVarAddrNode(lvaSwiftErrorLocal, type), typeInfo(type));
```
(to match the fact that we recognize both pointers and byrefs for `SwiftError`)